app/controllers/collections_controller.rb

```diff
@@ -7,6 +7,7 @@ class CollectionsController < ApplicationController
   before_action :set_collection_form_from_cocina, only: %i[show edit]
   before_action :set_status, only: %i[show edit]
   before_action :set_presenter, only: %i[show edit]
+  before_action :set_search_term, only: %i[works]
```

Comment:
If this is only used here in one method then would be clearer if inline in the method.

Reply:
was on the fence about that one, had it that way at first, flipping it back...